src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -294,6 +295,21 @@ public void setRatedIfNotExceeded(ZonedDateTime exerciseDueDate, ZonedDateTime s
         this.rated = exerciseDueDate == null || submissionDate.isBefore(exerciseDueDate);
     }
 
+    /**
+     * Sets the result to rated if:
+     * - It was created by an instructor (SubmissionType)
+     * - OR if the submissionDate is <= the exerciseDueDate.
+     * @param exerciseDueDate date after which no normal submission is considered rated.
+     * @param submission to which the result belongs.
+     */
+    public void setRatedIfNotExceeded(ZonedDateTime exerciseDueDate, Submission submission) {
+        if (submission.getType() == SubmissionType.INSTRUCTOR || submission.getType() == SubmissionType.TEST) {
+            this.rated = true;
+        }
+        else
```

Comment:
please always use curly brackets

Reply:
I changed it! I also checked if there is a formatting rule for this in Spotless, but apparently not: https://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fcore%2Fformatter%2FDefaultCodeFormatterConstants.html